crates/sozo/ops/src/migration/mod.rs

```diff
@@ -139,7 +142,9 @@ where
 
         if let Some(migration_output) = migration_output {
             if !ws.config().offline() {
-                upload_metadata(ws, account, migration_output, txn_config).await?;
+                upload_metadata(ws, &account, migration_output.clone(), txn_config).await?;
+                let world = WorldContract::new(world_address, account);
+                auto_authorize(ws, &world, &txn_config, &local_manifest, &migration_output).await?;
```

Comment:
If this fails, we shouldn't abort the whole migration. Only a warning should be displayed to mention that the authorization transaction couldn't be send.

Reply:
good catch ser, fixed it.